third_party/ccid/src/src/Info.plist.src

```diff
@@ -51,7 +51,7 @@
 	-->
 
 	<key>ifdDriverOptions</key>
-	<string>0x0000</string>
```

Comment:
BTW it's not the first time I forget to reapply it after the uprev.
I'm starting to believe I should just write a script to do this change automatically during the build...